src/core/frameworks/detect.ts

```diff
@@ -42,6 +43,8 @@ export async function generateConfiguration(app?: DetectedFolder, api?: Detected
       logger.silly(`Built API location "${computedApiLocation}" does not match root API location ${api.rootPath}, which is not supported yet`);
     }
     config.apiLocation = removeTrailingPathSep(api.rootPath);
+    config.apiLanguage = api.frameworks[0].name; //todo:check if the detected framework is valid or not
```

Comment:
NOT NEEDED and should be removed! It's already applied with the line `api.frameworks.forEach((f) => (config = { ...config, ...f.config }));`